lib/context_adapter_helper.js

```diff
@@ -205,6 +217,71 @@ function isAttributeIncluded(attributes, attributeName) {
 }
 
 /**
+ * Validates the attribute values received
+ * @param {object} context The context for logging
+ * @param {string} entityType The entity type (Button or Service)
+ * @param {Array} attributes The attributes array
+ * @return {*} True if the attribute value is valid, false otherwise
+ */
+function isValidAttributeValues(context, entityType, attributes) {
```

Comment:
If you are fine with it, I'll leave it as it is for the time being and as you mention if new validations are required we can take the approach you suggest :) Thanks!